### ISSUE TITLE
Running Parallel RSpec fails for Geordi > 3.2 if no binstub is present

### ISSUE THREAD

**Issue by denzelem** [makandra/geordi#111]
In a0b82fefcd0b5727039496e6174d52d0ec7f3e88 we introduced a bug, where the commands are not split properly. When the `bin/rspec` file is not present, we run `Bundler.clean_system(['bundle exec rake', 'parallel:spec'])`.

https://github.com/makandra/geordi/blob/738cfd00896b75091fd0ecedfc7774c2ee79e13d/lib/geordi/util.rb#L57-L59

But actually we want to run `Bundler.clean_system(['bundle', 'exec', 'rake', 'parallel:spec'])` with the arguments from the RSpec class.

https://github.com/makandra/geordi/blob/master/lib/geordi/commands/rspec.rb#L17

The result looks like this:

```
% geordi rspec                                                                                                                                                                                 (dst/geordi_issue $)

# Running specs
> All specs at once (using parallel_tests)

x Specs failed.
```

The expected output would look like this:

```
% bundle exec rake 'parallel:spec'                                                                                                                                                             (dst/geordi_issue $)
Using recorded test runtime
1 processes for 1 specs, ~ 1 specs per process

Randomized with seed 37552
.

Finished in 0.00214 seconds (files took 2.47 seconds to load)
1 example, 0 failures

Randomized with seed 37552


1 example, 0 failures

Took 2 seconds
```

**Comment by codener** [comment]
I believe `geordi migrate` has the same issue:

```
rake aborted!
Don't know how to build task 'db:migrate parallel:prepare'
```

**Comment by denzelem** [comment]
@codener This is already fixed in 9e5fbc637c14e46919aa7572457059456db6429e.
